### DISCUSSION TITLE
How to change ui.select label runtime dynamically from the code

### DISCUSSION THREAD

**Discussion by vmlstandard** [zauberzeug/nicegui#3335] [Q&A] [answered]
### Question

I have two simple selects:

```py
select1 =  ui.select(label="Label 1", options=["Label One", "Label Two", "Label Three"], with_input=True,
              on_change=lambda e: ui.notify(e.value)).classes('w-40')

select2 =  ui.select(label="Label2", options=["Option One", "Option Two", "Option Three"], with_input=True,
              on_change=lambda e: ui.notify(e.value)).classes('w-40')
```

and want to change label of the select2 to the value of selected option of the select1

it feels like "bind_value"  could do something like this, but I'm new with NiceGUI and haven't figired it out yet...

Would appreciate your advice

**Comment by falkoschindler** [comment, marked as answer]
This is a bit advanced, because there is currently no `label` property in `ui.select`. But you can call the `.props()` method:
```py
select1.on_value_change(lambda e: select2.props(f'label="{e.value}"'))
```

Or you can bind against the private `_props` dictionary and call `.update()` whenever the value changes:
```py
select1.on_value_change(lambda e: select2.props(f'label="{e.value}"'))
```

**Reply by vmlstandard** [reply]
Thank you so much - it worked like a charm
